pkg/restapi/kms/operation/operations_cryptobox.go

```diff
@@ -91,19 +100,21 @@ func (o *Operation) easyHandler(rw http.ResponseWriter, req *http.Request) { //n
 //        200: easyOpenResp
 //    default: errorResp
 func (o *Operation) easyOpenHandler(rw http.ResponseWriter, req *http.Request) { //nolint:funlen // TODO refactor
-	ctx, span := tracer.Start(req.Context(), "easyOpenHandler")
+	ctx, span := o.tracer.Start(req.Context(), "easyOpenHandler")
```

Comment:
is the tracer logging any sensitive data? eg keys, decrypted ciphertext, etc.

Reply:
nope, it's more about timing

Reply:
also I plan to update that approach in the future - probably move it to the middleware
I don't like that instrumentation code messes with production code - it becomes really hard to read the business logic